lib/richmond/rtfm.rb

```diff
@@ -10,75 +10,77 @@ class RTFM
     include Richmond::Logging
 
     def initialize 
-      @record_pattern =/^\=begin richmond/i 
+      @record_pattern      =/^\=begin richmond/i
       @output_file_pattern = /output-file:\s+.*\s*/i
-      @end_record_pattern = /^\=end/i
-      @mode = :paused
-    end 
+      @end_record_pattern  = /^\=end/i
+      @mode                = :paused
+    end
```

Comment:
This is a personal preference, but whenever I have a series of left-right assignments, it seems easier to read if I like up the ```=```.  

Just wanted to throw it out that's ok to do that sort of thing... it's not Resharper will realign this code afterwards.